### PR TITLE
Vaurca Stuff

### PR DESCRIPTION
**Pull request by Dekserrrrrr**
-Adds new Vaurca accents
-Adds the grounds for a new Vaurca subspecies
-Adjusts Vaurca stats, heavily
-Adds custom biometrics to Vaurcae
-Moves Vaurcae organs around, primarily, putting the brain in the chest
-Adds a new Vaurca language, only accessible by Warriors
-Renames all things relating to "zo'ra" to "Xal" or "Xallian"

### CODE REVIEW

**Comment by ghost** [comment]
Reported: Bug making humans unselectable

File: code/modules/mob/living/carbon/human/species/station/vaurca/vaurca_subspecies.dm [hunk 1]
```diff
@@ -68,12 +63,22 @@
...
+	max_pulse = 190// Default 160
+	body_temperature = 321.15 // 48°C
+
+	age_max = 150
+	default_genders = list(FEMALE, MALE)
 	economic_modifier = 3
 
 	speech_sounds = list('sound/voice/hiss1.ogg','sound/voice/hiss2.ogg','sound/voice/hiss3.ogg','sound/voice/hiss4.ogg')
-	speech_chance = 50
+	speech_chance = 100
```
Comment (ghost) [inline code/modules/mob/living/carbon/human/species/station/vaurca/vaurca_subspecies.dm]: This will get annoying fast

**Comment by Dekserrrrrr** [comment]
Accents are still fucky, we'll sort it out later. I'm tired and want this PR to be over. Good to merge, bug accents aren't in yet.

**Comment by ghost** [comment]
We expressly state we are not willing to merge incomplete or otherwise buggy/experimental PRs, and this is included in it. Once your accents are either removed entirely (all bugged code) or made to work, we can merge this.

We will not merge known bad code.

**Review by ghost — CHANGES_REQUESTED** [review]
All bugged code must be removed or fixed.

**Comment by Dekserrrrrr** [comment]
Code was always BUGged, but never affects the players in-game.

**Comment by ghost** [comment]
Once you fix the conflicts and mark it ready for review, someone can look at this.

**Pull request closed without merging**